css/interface.css

```diff
@@ -11,7 +11,7 @@
 }
 
 .form-horizontal {
-  padding: 15px;
+  padding: 14px;
```

Comment:
Previous all UI tickets we verified on chrome browser. So we verified this on the chrome browser.

Reply:
@yogeshmsharma-xoriant Sounds like it wasn't tested properly.

@amolsuryawanshi47 There's no explanation on why this is necessary. It's an odd solution to me because `10px` or `15px` is the default padding we use (and a lot of it comes from the Bootstrap default). This looks like it's trying to solve a problem without finding the root cause of why `1px` specifically needs to be subtracted.